jipdate.py

```diff
@@ -227,7 +232,7 @@ def parse_status_file(jira, filename):
     # [SWG-28]
     # [LITE-32]
     # etc ...
-    regex = r"^\[([A-Z]+-\d+)\]\n$"
+    regex = r"^\[(.*)\]\n$"
```

Comment:
IMHO, looking for all available issues doesn't seem better than using the error code returned by jira which is quite explicit : "Issue Does Not Exist".